inst/rmarkdown/templates/acs_article/template.yaml

```diff
@@ -0,0 +1,4 @@
+name: ACS
```

Comment:
You should spell out American Chemical Society here rather than using the abbreviation